heat/core/tests/test_suites/basic_test.py

```diff
@@ -209,9 +293,50 @@ def assert_func_equal(
             else:
                 self.assertTrue(np.array_equal(ht_res._DNDarray__array.numpy(), np_res))
 
-    def _create_random_array(self, shape):
+    def _create_random_array(self, shape, dtype=np.float64, low=-10000, high=10000):
```

Comment:
can you note that this creates a numpy array? others might misjudge it by its name, maybe just change array to np_array? just a thought

Reply:
I changed the name to `__create_random_np_array`